### PR TITLE
Implement commands queue execution

### PR DESCRIPTION
**Pull request by MartinOndejka**

### CODE REVIEW

**Comment by L-as** [comment]
What does this do

**Comment by MartinOndejka** [comment]
> What does this do

Queues sending of commands, retries if it fails (5 times), if it fails because of nonce precondition, it refetches the nonce

File: src/app/zeko/sequencer/lib/executor.ml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+open Base
```
Comment (L-as) [inline src/app/zeko/sequencer/lib/executor.ml]: Why are you using Base?

Comment (MartinOndejka) [reply src/app/zeko/sequencer/lib/executor.ml]: Not needed, I've removed it

**Comment by L-as** [comment]
I just noticed, but please tell me when you change the contract code, no matter how trivial. Otherwise the version inside my head and the version in the repository go out of sync.

File: src/lib/transaction_protocol_state/transaction_protocol_state.mli [hunk 2]
```diff
@@ -5,7 +5,7 @@ module Block_data : sig
   module Stable : sig
     module V2 : sig
       type t = Mina_state.Protocol_state.Body.Value.Stable.V2.t
-      [@@deriving sexp]
```
Comment (L-as) [inline src/lib/transaction_protocol_state/transaction_protocol_state.mli:8]: You didn't need to add `yojson` for all of these. You could just have used `sexp`.

Comment (L-as) [reply src/lib/transaction_protocol_state/transaction_protocol_state.mli:8]: Small changes like these to the MIna repository make us more out of sync, and without explicit notes, it's hard to remember it's been changed. Ideally you would be working to upstream small changes like these, but in this case it's not actually necessary AFAICT.

**Pull request merged**